crates/bvs-registry/src/msg.rs

```diff
@@ -49,6 +49,13 @@ pub enum QueryMsg {
     #[returns(StatusResponse)]
     Status { service: String, operator: String },
 
+    #[returns(StatusResponse)]
+    StatusAtHeight {
+        service: String,
+        operator: String,
+        height: u64,
+    },
+
```

Comment:
```suggestion
    Status { service: String, operator: String, height: Option<u64> },

```

I think it make more sense to do this, instead of having two path into status, make it one query_msg with optional `height`.

Reply:
hmm okay make sense. wanted to minimise breaking change but think we can still afford it now. will update